hoomd/md/IntegratorTwoStep.cc

```diff
@@ -145,7 +149,7 @@ void IntegratorTwoStep::update(unsigned int timestep)
         m_prof->push("Integrate");
 
     // perform the second step of the integration on all groups
-    for (method = m_methods.begin(); method != m_methods.end(); ++method)
+    for (auto method = m_methods.begin(); method != m_methods.end(); ++method)
```

Comment:
Same as comment above. If you prefer this still just resolve the issues.

Reply:
It's not a preference - much of this code was written before C++11 existed and long before it was widely available.